api/desecapi/tests/test_user_management.py

```diff
@@ -961,8 +970,9 @@ def test_renew_domain_fresh_183_days(self):
         self.assertEqual(Domain.objects.get(pk=domain.pk).renewal_state, Domain.RenewalState.FRESH)
         self.assertLess(timezone.now() - Domain.objects.get(pk=domain.pk).renewal_changed, timedelta(seconds=1))
 
+        # Check that other domains aren't affected
         for domain in self.my_domains[1:]:
```

Comment:
assert len(self.my_domains[1:])? just in case ...

Reply:
done